yearn/prices/pps.py

```diff
@@ -0,0 +1,64 @@
+from brownie import chain
+
+from cachetools.func import ttl_cache
+from yearn.utils import Singleton
+from yearn.networks import Network
+from yearn.exceptions import UnsupportedNetwork
+from yearn.utils import contract
+import yearn.prices.magic as magic
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+networks = [ Network.Mainnet, Network.Fantom, Network.Arbitrum ]
+assets = {
+    Network.Mainnet: {
+      "0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab": 12429262,
```

Comment:
delete the whole module, we shouldn't manually fix accounting errors, exporter should surface data as is

here is a period of nulls for this address:
- 12430455 first none
- 12430661 last none